src/components/modals/DarModal.js

```diff
@@ -61,6 +59,22 @@ const requiresManualReview = (darDetails) => {
 const DarModal = (props) => {
   //NOTE: Modal should be simple (raw information should be passed in as props) in order to ensure plug and play use
   const {showModal, closeModal, darDetails, datasets, researcher} = props;
+  const [institution, setInstitution] = useState();
+
+  useEffect(() => {
+
+    const returnInstitution = async (researcher) => {
+      const institutionId = researcher.institutionId;
+      if (isNil(institutionId)) {
+        setInstitution('- -');
+      } else {
+        const institute = await Institution.getById(institutionId);
```

Comment:
The fact that we need to do this means we should instead be updating the API that gets a researcher to get the full institution alongside all other properties. Can you add a tech-debt story for this? The API we need to update is `/api/user/{userId}`. The UI work will be to try and remove any unnecessary calls to `Institution.getById(institutionId)` once the full institution is part of the user response.